integration-cli/docker_cli_kill_test.go

```diff
@@ -42,9 +42,11 @@ func (s *DockerSuite) TestKillDifferentUserContainer(c *check.C) {
 	dockerCmd(c, "kill", cleanedContainerID)
 
 	out, _ = dockerCmd(c, "ps", "-q")
-	if strings.Contains(out, cleanedContainerID) {
-		c.Fatal("killed container is still running")
-	}
+	processContainerID := strings.TrimSpace(out)
+
+	c.Assert(waitRun(processContainerID), check.IsNil)
```

Comment:
This line and the line above are not right (and not needed) and make the build fail.